src/runtime/virtcontainers/qemu_s390x.go

```diff
@@ -424,3 +424,13 @@ func (q *qemuS390x) qomGetPciPath(qemuID string, qmpCh *qmpChannel) (types.PciPa
 	hvLogger.Warnf("qomGetPciPath not implemented for s390x")
 	return types.PciPath{}, nil
 }
+
+func (q *qemuS390x) memoryTopology(memoryMb, hostMemoryMb uint64, slots uint8) govmmQemu.Memory {
+	// For no hotplug memory, set hostMemoryMb to memoryMb, otherwise will cause
+	// `invalid value of maxmem: maximum memory size (0x0) must be at least the initial memory size`
+	if hostMemoryMb == 0 {
+		hostMemoryMb = memoryMb
```

Comment:
if you pass a non-zero `hostMemoryMb` to `genericMemoryTopology()` you won't get the expected output. See my original PR. The goal is to get the memory size completely without `maxmem=`. Or is this needed for s390x?